docs(community): reposition community and support docs to reflect both products (#77661)
airbytehq/airbyte: docs/community/contributing-to-airbyte/README.md

# Contributing to Airbyte

> Thank you for your interest in contributing! Airbyte accepts community contributions to its open-source data replication connectors in the [`airbyte`](https://github.com/airbytehq/airbyte) repository. Airbyte Agents is a separate closed-source product and does not accept community code contributions.

Author: ```suggestion
Thank you for your interest in contributing! Airbyte accepts community contributions to its open-source data replication connectors in the [`airbyte`](https://github.com/airbytehq/airbyte) repository. Airbyte Agents does not accept community code contributions.
```

> We appreciate first-time contributors and are happy to help you get started. Join our [community Slack](https://slack.airbyte.io) and feel free to reach out with questions in [`#help-connector-development` channel](https://airbytehq.slack.com/archives/C027KKE4BCZ).

Reviewer: ```suggestion
We appreciate first-time contributors and are happy to help you get started. Join our [Airbyte Community Slack](https://slack.airbyte.io) and feel free to reach out with questions in [`#help-connector-development` channel](https://airbytehq.slack.com/archives/C027KKE4BCZ).
```

> We appreciate first-time contributors and are happy to help you get started. Join our [community Slack](https://slack.airbyte.io) and feel free to reach out with questions in [`#dev-and-contribuions` channel](https://airbytehq.slack.com/archives/C054V9JFTC6).

Author: ```suggestion
We appreciate first-time contributors and are happy to help you get started. Join our [community Slack](https://slack.airbyte.io) and feel free to reach out with questions in [`#help-connector-development` channel](https://airbytehq.slack.com/archives/C027KKE4BCZ).
```

### Contributions we don't accept

> - Platform contributions. In mid-2025, Airbyte stopped accepting community contributions to the Airbyte platform. Continue reporting issues through GitHub so we can investigate and prioritize fixes and improvements.

Author: ```suggestion
- Platform contributions. In mid-2025, Airbyte stopped accepting community contributions to the data replication platform. Continue reporting issues through GitHub so we can investigate and prioritize fixes and improvements.
```

> - Platform contributions. In mid-2025, Airbyte stopped accepting community contributions to the data replication platform. Continue reporting issues through GitHub so we can investigate and prioritize fixes and improvements.

Author: ```suggestion
- Data replication platform contributions: in mid-2025, Airbyte stopped accepting community contributions to the data replication platform. Continue reporting issues through GitHub so we can investigate and prioritize fixes and improvements.
```

> - Data replication platform contributions: in mid-2025, Airbyte stopped accepting community contributions to the data replication platform. Continue reporting issues through GitHub so we can investigate and prioritize fixes and improvements.

Reviewer: Should we be realistic and say that we arent accepting destination contributions either? Or really db source contributions?

Author: Devin: Add a line that says "Contributions to destination connectors. We only accept contributions to source connectors."

> - Airbyte Agents contributions. Airbyte Agents is closed-source. For bugs or feature requests related to Airbyte Agents, use the [support center](https://support.airbyte.com/hc/en-us/requests/new).

Author: ```suggestion
- Airbyte Agents contributions: for bugs or feature requests related to Airbyte Agents, if you're a paid customer, use the [support center](https://support.airbyte.com/hc/en-us/requests/new), and if you're a free user, use the [#airbyte-agents](https://airbytehq.slack.com/archives/C0B1CUSEQKT) channel in Slack.
```

> - Airbyte Agents contributions: for bugs or feature requests related to Airbyte Agents, if you're a paid customer, use the [support center](https://support.airbyte.com/hc/en-us/requests/new), and if you're a free user, use the [`#airbyte-agents`](https://airbytehq.slack.com/archives/C0B1CUSEQKT) channel in Slack.

Reviewer: ```suggestion
- Airbyte Agents contributions: for bugs or feature requests related to Airbyte Agents, if you're a paid customer, use the [Support Center](https://support.airbyte.com/hc/en-us/requests/new), and if you're a free user, use the [`#airbyte-agents`](https://airbytehq.slack.com/archives/C0B1CUSEQKT) channel in Slack.
```
